src/AutoAffix.js

```diff
@@ -119,12 +133,12 @@ class AutoAffix extends React.Component {
         <div ref="positioner" />
 
         <Affix
+          {...props}
```

Comment:
Oops.

I think I wanted to let the user override `offsetTop` or something via `props`, but on second thought that's a silly thing to do.